### PR TITLE
Swap getUser auth test fixes to an earlier patch version

### PR DESCRIPTION
**Pull request by alexneyman-MSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

This change updates the versions which we should be testing the new getUser tests on. We've pushed it back because we're doing a patch release. 

### Main changes in the PR:

1. <Change 1>
2. <Change 2>

### Validation performed:

1. <Step 1>
2. <Step 2>

### Unit Tests added:

> Unit tests are required for all changes. If no unit tests were added as part of this change, please explain why they aren't necessary.

<Yes/No>

### End-to-end tests added:

<Yes/No>

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

<Yes/No>

### Related PRs:

> Remove this section if n/a

### Next/remaining steps:

> List the next or remaining steps in implementing the overall feature in subsequent PRs (or is the feature 100% complete after this?).

> Remove this section if n/a

- [ ] Item 1
- [ ] Item 2

### Screenshots:

> Remove this section if n/a

| Before     | After      |
| ---------- | ---------- |
| < image1 > | < image2 > |

### CODE REVIEW

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
       "title": "getUser API Call - Failure (Not full trust and not Microsoft owned)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": ">3.1.0"
+        "web": ">3.0.2"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:37]: `>3.0.2 && <3.1 || > 3.1`

Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:37]:
```suggestion
        "web": ">3.0.2 && <3.1 || > 3.1"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
       "title": "getUser API Call - Success (Full trust app)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": ">3.1.0"
+        "web": ">3.0.2"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:49]:
```suggestion
        "web": ">3.0.2 && <3.1 || > 3.1"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 3]
```diff
@@ -58,7 +58,7 @@
       "title": "getUser API Call - Success (Full trust and Microsoft owned app)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": ">3.1.0"
+        "web": ">3.0.2"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:61]:
```suggestion
        "web": ">3.0.2 && <3.1 || > 3.1"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 4]
```diff
@@ -22,7 +22,7 @@
       "title": "getUser API Call - Failure (Not full trust)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": "<=3.1.0"
+        "web": "<=3.0.2"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:25]:
```suggestion
        "web": ">3.0.2 && <3.1 || > 3.1"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 5]
```diff
@@ -22,7 +22,7 @@
       "title": "getUser API Call - Failure (Not full trust)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": "<=3.1.0"
+        "web": ">3.0.2 && <3.1 || > 3.1"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:25]:
```suggestion
        "web": ">3.0.2  <3.1 || > 3.1.0"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 6]
```diff
@@ -34,7 +34,7 @@
       "title": "getUser API Call - Failure (Not full trust and not Microsoft owned)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": ">3.1.0"
+        "web": ">3.0.2 && <3.1 || > 3.1"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:37]:
```suggestion
        "web": ">3.0.2  <3.1 || > 3.1.0"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 7]
```diff
@@ -46,7 +46,7 @@
       "title": "getUser API Call - Success (Full trust app)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": ">3.1.0"
+        "web": ">3.0.2 && <3.1 || > 3.1"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:49]:
```suggestion
        "web": ">3.0.2  <3.1 || > 3.1.0"
```

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 8]
```diff
@@ -58,7 +58,7 @@
       "title": "getUser API Call - Success (Full trust and Microsoft owned app)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": ">3.1.0"
+        "web": ">3.0.2 && <3.1 || > 3.1"
```
Comment (shrshindeMSFT) [inline apps/teams-test-app/e2e-test-data/authentication.json:61]:
```suggestion
        "web": ">3.0.2  <3.1 || > 3.1.0"
```

**Review by shrshindeMSFT — DISMISSED** [review]

**Review by shrshindeMSFT — DISMISSED** [review]

**Review by erinha — DISMISSED** [review]

File: apps/teams-test-app/e2e-test-data/authentication.json [hunk 9]
```diff
@@ -22,7 +22,7 @@
       "title": "getUser API Call - Failure (Not full trust)",
       "version": ">2.25.0",
       "hostSdkVersion": {
-        "web": "<=3.1.0"
+        "web": ">3.0.2"
```
Comment (erinha) [inline apps/teams-test-app/e2e-test-data/authentication.json:25]: This should be <=3.0.2

Comment (erinha) [inline apps/teams-test-app/e2e-test-data/authentication.json:25]:
```suggestion
        "web": "<=3.0.2"
```

**Review by shrshindeMSFT — APPROVED** [review]

**Review by erinha — COMMENTED** [review]
🚀

**Review by erinha — APPROVED** [review]

**Review by sunbrk — APPROVED** [review]

**Pull request merged**